templates/torrc.j2

```diff
@@ -60,7 +60,13 @@
 #CookieAuthentication 1
 
 {% for key, value in hidden_services_configuration.iteritems() %}
+{% if key == 'HidServAuth' %}
```

Comment:
There are multiple values that can be repeated in the torrc—consider adding a list var that stores the names of those options, so _all_ list-friendly config options can be iterated over. Not adding in this PR because it seems out of scope.